contracts/test/integration/IntegrationTestPCVGuardian.t.sol

```diff
@@ -26,7 +26,7 @@ contract IntegrationTestPCVGuardian is DSTest {
     IVolt private fei = IVolt(0x956F47F50A910163D8BF957Cf5846D573E7f87CA);
```

Comment:
I think I really like this approach, adding more addresses to the current test addresses setup will run into a stack to deep error, and will need some work around for that at some point, so I think this flow is cleaner & also solves for that

Reply:
Let's try out this approach and see if we can get around the stack too deep error by using a library constant instead of the current setup. https://github.com/fei-protocol/fei-protocol-core/blob/develop/contracts/test/integration/fixtures/MainnetAddresses.sol